### ISSUE TITLE
Output Decoding

### ISSUE THREAD

**Issue by muellerzr** [muellerzr/fastinference_pytorch#12]
- [ ] Store loss function used (`fastinference`)
- [ ] Cross Entropy Loss
- [ ] BCELossLogits
- [ ] Example of Custom
